pkg/retriever/reportretriever.go

```diff
@@ -232,11 +254,12 @@ func (r *Retriever) GetPolicyInfo(
 	responseBody types.ResponseBody,
 	cluster types.ManagedClusterInfo,
 ) (types.ProcessorData, error) {
-	glog.Infof("Creating Policy Info for cluster %s (%s)", cluster.Namespace, cluster.ClusterID)
+	glog.V(2).Infof("Starting GetPolicyInfo for cluster %s (%s)", cluster.Namespace, cluster.ClusterID)
 	reports := types.Reports{}
 	for _, clusterErrored := range responseBody.Errors {
-        glog.Warningf("Error occured while requesting Insights for cluster: %s", clusterErrored)
-    }
+		glog.Warningf("No Reports returned from CCX Insights for cluster: %s", clusterErrored)
+		glog.V(2).Infof("Errors returned from CCX Insights for cluster : %v", responseBody)
```

Comment:
This one doesnt quite make sense to me. `for cluster: %v`.. you are using responseBody for the value not a cluster id/name